include/fastdds/dds/core/policy/ParameterTypes.hpp

```diff
@@ -1207,19 +1253,23 @@ class ParameterPropertyList_t : public Parameter_t
 
     /**
      * Constructor using a parameter PID and the parameter length
+     *
+     * @param pid Pid of the parameter
      * @param in_length Its associated length
      */
     ParameterPropertyList_t(
-            ParameterId_t /*pid*/,
+            ParameterId_t pid,
             uint16_t in_length)
         : Parameter_t(PID_PROPERTY_LIST, in_length)
         , Nproperties_ (0)
         , limit_size_ (false)
     {
+        (void)pid;
```

Comment:
```suggestion
        static_cast<void>(pid);
```

We usually try to use C++ syntax instead of C one.